docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -557,9 +557,24 @@ which shows (p. 31) how people in rural counties are hard to match
 from 2010 Decennial Census
 <https://www.census.gov/newsroom/blogs/director/2010/02/the-four-principal-ways-we-conduct-the-census.html>`_
 there is 9% of the US population where the mail is not delivered to
-the residence uniformly.  For these households, we might want to capture
-different addresses in the decennial census simulated output and the
-tax return simulated output.
+the residence uniformly.  For these households, we might want to
+capture different addresses in the decennial census simulated output
+and the tax return simulated output.  We can represent this by
+mantaining a *mailing address* for each household that is sometimes
```

Comment:
In our initial implementation we are not including this. Would you like us to?

Reply:
not necess yet